package.json

```diff
@@ -2,21 +2,24 @@
   "name": "@ebay/ebayui-core",
   "version": "0.1.0-1",
   "scripts": {
+    "installMarkoV3": "yarn add marko@^3 marko-widgets@^6 -D",
```

Comment:
@yomed what are your thoughts on just having multiple versions of Marko as devDependencies?

IE:
```
"devDependencies": {
  ...
    "marko3": "npm:marko@3",
    "marko4": "npm:marko@4"
  ...
}
```

Which enables `require('marko3')` and `require('marko4'). We could then setup the tests to create a symlink before running which feels a bit cleaner to me than installing before running the tests, especially outside of the CI.

Thoughts?

Reply:
I agree that that would be better -- that was actually my first attempt when I introduced the integration tests. It sort of worked (to begin with), but I ended up running into a lot of strange errors which I believe were related to Marko implicitly depending on there only being a single version installed. I don't remember the exact details, but I think some of them were around the Marko compiler relying on certain defined globals, which would get messed up with multiple versions. Maybe we can look to improve this in Marko in the future?

Reply:
Alright. For now I don't think it's worth holding back this PR but this could be an enhancement we look into in the future.

Otherwise LGTM.